app/assets/stylesheets/views/_help-page.scss

```diff
@@ -0,0 +1,9 @@
+.help-page {
+  .add-title-margin {
+    @include responsive-top-margin;
+  }
+
+  .meta-data {
```

Comment:
Could this be `last-updated` too? `meta-data` risks being confused with the metadata component.